consensus/polybft/polybftcontracts/package.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Is there a ticket to move this to a submodule?

Reply:
Yep, we are going to obsolete this entire `polybftcontracts` package once this issue is addressed: https://polygon.atlassian.net/browse/EVM-93.